src/content/api/businesses.mdx

```diff
@@ -0,0 +1,417 @@
+---
+title: Businesses
+description: Business model and related endpoints.
+draft: true
+nav:
+  path: API
+  order: 9
+---
+import Properties from '../../components/Properties.astro';
```

Comment:
I wonder if there is a way we don't need to import these components in every markdown file

Reply:
@AndyClifford I've been wondering that too 🤔 
Couldn't see anything in the Astro docs about it.